samples/v1.5/Test/TableColumnSetComparison.json

```diff
@@ -0,0 +1,1430 @@
+{
+    "type": "AdaptiveCard",
```

Comment:
tabs?

Reply:
Finally figured out the issue. I didn't have "format on paste" enabled.